runtime/ms_rest_azure/lib/ms_rest_azure/azure_service_client.rb

```diff
@@ -19,6 +19,8 @@ def initialize(credentials, options = nil)
       @request_headers =  {
           'Content-Type' => 'application/json;charset=utf-8' # This is the current default for Azure services, and content-type supported by Autorest
       }
+      add_user_agent_information("ms_rest_azure/#{MsRestAzure::VERSION}")
+      add_user_agent_information("Azure-SDK-For-Ruby")
```

Comment:
Yes, we are including "Azure-SDK-For-Ruby" string to make sure current telemetry system works as well as that is the defined string for each language "Azure-SDK-For-{LANG}" for telemetry collection. So the purpose it two-fold 1. Keep existing telemetry alive and 2. That is the current way to distinguish between languages. 

Regarding keeping "Azure-SDK-For-Ruby" on ms_rest_azure i am not sure what could be the best decision because that actually represents the SDK and should go into each management client but on the other hand keeping it in each client seems overhead. As we are the one going to interpret the data, I am thinking it'd be fine to put that string in ms_rest_azure. What do you think?